test/cmd.js

```diff
@@ -14,12 +14,12 @@ var TEMP_DIR = path.resolve(__dirname, '..', 'temp', String(process.pid + Math.r
 
 describe('express(1)', function () {
   before(function (done) {
-    this.timeout(30000);
+    this.timeout(60000);
     cleanup(done);
   });
 
   after(function (done) {
-    this.timeout(30000);
+    this.timeout(120000);
```

Comment:
These timeouts are large as they are and now even larger. Please revert them back.